monkey/infection_monkey/pe/actions/__init__.py

```diff
@@ -0,0 +1,27 @@
+from abc import ABCMeta, abstractmethod
+from os.path import dirname, basename, isfile, join, realpath
+import glob
+
+__author__ = 'D3fa1t'
+
+
+class HostPrivExploiter(object):
+    __metaclass__ = ABCMeta
+
+    @abstractmethod
+    def try_priv_esc(self, commad):
+        raise NotImplementedError()
+
+    def send_pe_telemetry(self, result, local_ip):
+        from infection_monkey.control import ControlClient
+        ControlClient.send_telemetry('pe', {'result': result, 'pe_name': self.__class__.__name__, 'ip': local_ip})
+
+
+def get_pe_files():
+    """
+        Gets all files under current directory(/actions)
+        :return: list of all files without .py ending
+    """
```

Comment:
Same as in PBA, which means we'll have to refactor sooner or later. It'll be my job probably.